components/HelpfulHoursTab/AcademicOffice.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState} from 'react';
+import { View, FlatList, Text, Linking} from 'react-native';
+import { Card, List, Title, Subheading } from 'react-native-paper';
+
+// import Firebase Relative
+import { db } from '../../firebase/config';
+import { doc, getDoc } from 'firebase/firestore';
+
+// import StyleSheet 
+import styles from './styles';
+
+import { AntDesign } from "@expo/vector-icons";
+
+const AcademicOffice = () => {
+    const [academicOffice, setAcademicOffice] = useState([]);
+    // useState for List Accordions
+    const [expanded, setExpanded] = useState(false); // set this to false, don't want to open automatically
+    const handlePress = () => setExpanded(!expanded);
+
+    useEffect(() => {
+        // get AcademicOffice data from firestore
+        const docRef = doc(db, "helpful hours" , "academic office");
+        getDoc(docRef) // get the document
+            .then((doc) => {
+                setAcademicOffice(doc.data().sections);
+            })
+            .catch((err) => {
+                console.log(err);
+            })
+    }, []);
+
+    return (
+        <View style = {styles.contentContainer}>
+        <List.Accordion 
+            children = {{children: 0}}
+            style = {styles.accordion}
+            title = "Academic Office"
```

Comment:
It looks like the files under components/HelpfulHoursTab all look more or less the same. Let's make the code more reusable.